tt_eager/tt_dnn/op_library/conv/multi_core_optimized_conv_sharded/optimized_conv_op_sharded_v2.cpp

```diff
@@ -1043,8 +1047,6 @@ operation::ProgramWithCallbacks multi_core_optimized_conv_sharded_v2_(const Tens
                 window_outer, // window_outer
                 window_inner, // window_inner = 9 / 3, ie. read 3 width coalesced
 
-                (uint32_t) noop_core,
-
```

Comment:
I think this arg needs to remain, else the subsequent indexing in the kernel would go wrong.

Reply:
Will revert.